### DISCUSSION TITLE
这东西怎么用

### DISCUSSION THREAD

**Discussion by xinwei-good** [gaoyia/bilibili-uwp-download-fixtool#5] [Q&A] [answered]
怎么去配置兼容版本环境

**Comment by gaoyia** [comment, marked as answer]
暂时只支持windows下使用。 ，从release里面下载，源码没有写electron打包脚本。只写了核心代码手动打包的。自己用用懒得搞。

打开软件从上往下，选择你用哔哩哔哩UWP下载的视频文件夹一步步操作即可。视频就可以用第三方插件播放了。一次不要处理太多文件可能会卡死（没有做优化）
